src/lang_stats.cpp

```diff
@@ -0,0 +1,17 @@
+#include <initializer_list>
+#include <lang_stats.h>
+
+static constexpr std::initializer_list<lang_stats> all_lang_stats = {
+#include <lang_stats.inc>
+};
```

Comment:
GCC 12 doesn't like this.
```gdb
../cataclysm-dda-git/src/lang_stats.cpp:32:1: error: modification of ‘<temporary>’ is not a constant expression
   32 | };
      | ^

```

`static constexpr std::array<lang_stats,27> all_lang_stats = {` works though, with `lang_stats{"ar", 5, 85479},` and so on.

Reply:
Thanks.  I found a different workaround that doesn't require explicitly hardcoding the number of languages.